integrations/database/mariadb/install.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit -o errtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+NAMESPACE="mariadb"
+
+# Install MariaDB operator.
+kubectl create ns mariadb-operator --dry-run=client -o yaml | kubectl apply -f -
+helm upgrade --install --namespace mariadb-operator mariadb-operator mariadb-operator/mariadb-operator
+kubectl -n mariadb-operator rollout status deployment --timeout=90s
+
+# Create the MariaDB database server.
```

Comment:
Since there are waits here, you should echo what the script is doing so anyone watching doesn't have to translate the commands.

```suggestion
echo Create the MariaDB database server.
```

Reply:
Fixed.